index.html

```diff
@@ -68,17 +69,29 @@ <h4>Unit</h4>
           </div>
         </form>
       </div>
-      <div id="more-info-page" class="hidden"></div>
+      <div id="more-info-page" class="hidden">
+        <p>hello world</p>
```

Comment:
Probably shouldn't leave "hello world" here, the text should be more descriptive of what's actually being shown

Reply:
The p element was leftover from debugging getting the more-info-page to show when making sure view swapping was working.